server/gokbg3/grails-app/services/org/gokb/FTUpdateService.groovy

```diff
@@ -389,6 +398,76 @@ class FTUpdateService {
         }
 
         result.componentType = kbc.class.simpleName
+        result.tippTitleMedium = kbc.title ? kbc.title.medium : ""
+
+        result.accessStartDate = kbc.accessStartDate ? kbc.accessStartDate.toString() : ""
+        result.accessEndDate = kbc.accessEndDate ? kbc.accessEndDate.toString() : ""
+
+        result.subjectArea = kbc.subjectArea ?: ""
+        result.series = kbc.series ?: ""
+
+        if (kbc.title?.niceName == 'Book'){
+
+          // edition for eBooks
+          def edition = [:]
+          if (kbc.title?.editionNumber){
+            edition.number = kbc.title.editionNumber
+          }
+          if (kbc.title?.editionDifferentiator){
+            edition.differentiator = kbc.title.editionDifferentiator
+          }
+          if (kbc.title?.editionStatement){
+            edition.statement = kbc.title.editionStatement
+          }
+          if (!edition.isEmpty()){
+            result.titleEdition = edition
+          }
+
+          // simple eBook fields
+          result.titleVolumeNumber = kbc.title?.volumeNumber ?: ""
+          result.titleDateFirstInPrint = kbc.title?.dateFirstInPrint ?: ""
+          result.titleDateFirstOnline = kbc.title?.dateFirstOnline ?: ""
+          result.titleFirstEditor = kbc.title?.firstEditor ?: ""
+          result.titleFirstAuthor = kbc.title?.firstAuthor ?: ""
+          result.titleImprint = kbc.title?.imprint ?: ""
```

Comment:
An imprint of a title would be an Imprint class object, so `ìmprint?.name` would be better.